Source/SmallBasic.Editor/Components/Pages/Debug/MemoryExplorer.scss

```diff
@@ -249,7 +249,7 @@ memory-explorer {
             @extend base-member;
             flex-shrink: 0;
             flex-grow: 0;
-            padding-bottom: 15px;
+            margin-bottom: 50px;
```

Comment:
Why changing that value? this is responsible for this orange margin:

![image](https://user-images.githubusercontent.com/15987992/55780889-d7c54980-5a5d-11e9-9c47-b7612f64219d.png)